bcda/api/v1/api.go

```diff
@@ -285,6 +293,38 @@ func ServeData(w http.ResponseWriter, r *http.Request) {
 	} else {
 		http.ServeFile(w, r, fmt.Sprintf("%s/%s/%s", dataDir, jobID, fileName))
 	}
+
+	// Logging request for auditing
+	var jobIdInt, err = strconv.ParseUint(jobID, 10, 64)
+
+	if err == nil {
+		var jobKeys, err = repository.GetJobKeys(context.Background(), uint(jobIdInt))
```

Comment:
what do you think about adding a new function to repository with the following signature to encapsulate some of this logic?  `GetJobKey(ctx context.Context, jobID uint, fileName string) (*models.JobKey, error)`

Reply:
I think it will be easier to write a test case for that if it stands alone as its own func